src/test/resources/refaster/PicnicRulesRecipes.java

```diff
@@ -52,37 +52,205 @@ public String getDisplayName() {
 
     @Override
     public String getDescription() {
-        return "Refaster template recipes for `foo.PicnicRules`. [Source](https://error-prone.picnic.tech/refasterrules/PicnicRules).";
+        return "Picnic rules for refaster, showing how JavaDoc is converted to Markdown [Source](https://error-prone.picnic.tech/refasterrules/PicnicRules).";
     }
 
     @Override
     public List<Recipe> getRecipeList() {
         return Arrays.asList(
-                new NestedRuleRecipe()
+                new FirstRuleRecipe(),
+                new SecondRuleRecipe(),
+                new ThirdRuleRecipe(),
+                new FourthRuleRecipe()
         );
     }
 
     /**
-     * OpenRewrite recipe created for Refaster template {@code PicnicRules.NestedRule}.
+     * OpenRewrite recipe created for Refaster template {@code PicnicRules.FirstRule}.
      */
     @SuppressWarnings("all")
     @NullMarked
     @Generated("org.openrewrite.java.template.processor.RefasterTemplateProcessor")
-    public static class NestedRuleRecipe extends Recipe {
+    public static class FirstRuleRecipe extends Recipe {
 
         /**
          * Instantiates a new instance.
          */
-        public NestedRuleRecipe() {}
+        public FirstRuleRecipe() {}
 
         @Override
         public String getDisplayName() {
-            return "Refaster template `PicnicRules.NestedRule`";
+            return "Refaster template `PicnicRules.FirstRule`";
         }
 
         @Override
         public String getDescription() {
-            return "Recipe created for the following Refaster template:\n```java\npublic static class NestedRule {\n    \n    @BeforeTemplate()\n    String before(String s, String s1, String s2) {\n        return s.replaceAll(s1, s2);\n    }\n    \n    @AfterTemplate()\n    String after(String s, String s1, String s2) {\n        return s != null ? s.replaceAll(s1, s2) : s;\n    }\n}\n```\n.";
+            return "A single line used as description";
```

Comment:
This generated block greatly inflates our docs, which hurts readability. Better then to use the JavaDoc provided instead.